lib/ood_core/job/adapters/helper.rb

```diff
@@ -12,6 +12,23 @@ class Helper
         def self.bin_path(cmd, bin_default, bin_overrides)
           bin_overrides.fetch(cmd.to_s) { Pathname.new(bin_default.to_s).join(cmd.to_s).to_s }
         end
+        
+        # Gets a command that submits command on another host via ssh
+        # @param submit_host [String] where to submit the command
+        # @param cmd [String] the desired command to execute on another host
+        # @param cmd_args [Array] arguments to the command specified above
+        #
+        # @return cmd [String] command wrapped in ssh if submit_host is present
+        # @return args [Array] command arguments including ssh_flags and original command
+        def self.ssh_wrap(submit_host, cmd, cmd_args)
+          return cmd, cmd_args if submit_host.empty?
```

Comment:
nil.empty? crashes. Protect against nil by coercing to string, since nil.to_s == "":

```
return cmd, cmd_args if submit_host.to_s.empty?
```